github-dark.css

```diff
@@ -1989,7 +1989,7 @@
   }
   .text-pending, a.text-pending, .compare-pr-placeholder p,
   .compare-pr-placeholder .help-link, .text-pending strong {
-    color: #cc5 !important;
+    color: #ddd !important;
   }
```

Comment:
The reason I hadn't changed these colors is because I like to get my eyes on what is changing. In some cases, like the travis status we don't want to change the yellow.